src/Dashboard/ApiControllers/FunctionsController.cs

```diff
@@ -341,6 +341,7 @@ private InvocationLogViewModel CreateInvocationEntry(RecentInvocationEntry entry
             metadataSnapshot.EndTime = entry.EndTime;
             metadataSnapshot.Succeeded = entry.Succeeded;
             metadataSnapshot.Heartbeat = entry.Heartbeat;
+            metadataSnapshot.HeartbeatExpiredTime = entry.HeartbeatExpiredTime;
```

Comment:
I'll do the rename & comment.  This is not persisted data, so we're not painting ourselves into a corner. 

---
In reply to: [90485175](https://github.com/Azure/azure-webjobs-sdk/pull/928#discussion_r90485175) [](ancestors = 90485175)